packages/legend-application-studio/src/stores/EditorGraphState.ts

```diff
@@ -666,6 +666,13 @@ export class EditorGraphState {
           return FormModeCompilationOutcome.FAILED;
         }
         this.editorStore.setGraphEditMode(GRAPH_EDITOR_MODE.GRAMMAR_TEXT);
```

Comment:
should we maybe have a function to convert to grammar test, and sensure we we do all the required changes when we switch over.

Reply:
that would make more sense